test/index.js

```diff
@@ -47,7 +47,7 @@ internals.cleanup = (session, done) => {
                 Fs.unlinkSync(filePath);
             });
 
-        session.knex.destroy().asCallback(done);
+        session.knex.destroy().then(() => done(), done);
```

Comment:
What lead to this change? I thought asCallback would pass the error to done and we'd be able to get it from there. If destroy() fails what happens now?

Reply:
You're totally right—upgrading knex broke this https://github.com/tgriesser/knex/pull/2589